Find-and-Replace/Replace-text-with-data-from-database/README.md

```diff
@@ -0,0 +1,18 @@
+# How to replace text with data from database
+This example shows how to find a text in a Word document and replace it with a text from the database.
+
+# How to run this Sample
+
+1. Download this project to a location in your disk.
+
+2. Create database with column "TextToFind" and "ReplacementText" as the screenshot shown below.
```

Comment:
Create database with column "TextToFind" and "ReplacementText" as like the screenshot shown below.

Reply:
@MohanaselvamJothi -I have modified it